<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-84VHTLC" failures="1" tests="2" name="Command line test" time="280.229" errors="0" timestamp="30 Nov 2019 19:54:55 GMT">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="feature" time="171.27">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.AssertionError: Warning message displayed expected [true] but found [false]">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: Warning message displayed expected [true] but found [false]
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
at applitools.testrunner.TestRunner.testRunner(TestRunner.java:42)
at applitools.testrunner.TestRunner.executeTest(TestRunner.java:29)
Caused by: java.lang.AssertionError: Warning message displayed expected [true] but found [false]
at applitools.utils.AssertUtils.assertTrue(AssertUtils.java:27)
at applitools.stepdefinitions.TraditionalTestStepDefinitions.verify_no_credentials_warning_message(TraditionalTestStepDefinitions.java:57)
at âœ½.Then Verify No Credentials Warning message(TraditionalTest.feature:11)
... Removed 50 stack frames]]>
    </failure>
  </testcase> <!-- feature -->
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="feature" time="107.37"/>
</testsuite> <!-- Command line test -->
